src/hooks/useLoginForm.ts

```diff
@@ -0,0 +1,45 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import * as z from "zod";
+
+// Data for checking validate login form
+const dataDummy = {
+  id: 1,
+  email: "dummy@gmail.com",
+  password: "Admin123",
+};
+
+const formSchema = z.object({
+  email: z.string().email({ message: "Invalid email address" }),
+  password: z.string().min(1, { message: "Invalid password" }),
+});
+
+export type LoginFormValues = z.infer<typeof formSchema>;
+
+const useLoginForm = (): any => {
+  const [loginError, setLoginError] = useState<string | null>(null);
+  const form = useForm<LoginFormValues>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+
+  function onSubmit(values: LoginFormValues): void {
+    if (
+      values.email === dataDummy.email &&
+      values.password === dataDummy.password
+    ) {
+      setLoginError(null);
+      alert("Login successful!");
+    } else {
+      setLoginError("Email or password is incorrect");
+    }
```

Comment:
Set an error when the email and password doesn't match using `form.setError()` with the message 'Wrong password'. Navigate to `/admin/list-anggota` when success

Reply:
Misi mas, aku izin mau beri saran kek nya klo message nya "Email or password is incorrect" itu secara ux lebih enak soalnya buat jagain kalo ada case emng yang salah bukan dari password nya doang, jadi si user tetep ngecek emailnya ntah itu typo atau ga. Misal lagi worst case nya password nya udh bener tpi email nya salah, buat input button password nya kan di kita kan gd fitur show password, jadi si user ntar malah bingung sendiri dia kira password yg dia masukin salah.